swagger/swagger-invocation/invocation-core/src/main/java/org/apache/servicecomb/swagger/engine/SwaggerProducerOperation.java

```diff
@@ -153,7 +152,7 @@ public void doCompletableFutureInvoke(SwaggerInvocation invocation, AsyncRespons
     } catch (Throwable e) {
       if (shouldPrintErrorLog(e)) {
         LOGGER.error("unexpected error operation={}, message={}",
-            invocation.getInvocationQualifiedName(), ExceptionUtils.getExceptionMessageWithoutTrace(e));
+            invocation.getInvocationQualifiedName(), e.getMessage());
```

Comment:
Why modify this ?  There is no SCB-2387 for this.

Reply:
[This code will print too much sensitive information when JSON parsing fails](https://github.com/apache/servicecomb-java-chassis/issues/2679)

Reply:
Not all of the sensitive contents can be blocked by this way. For example, the Jackson may put raw JSON string into exception message.

Reply:
Personally I'd prefer not modify this. This is not an actual serious information leak problem. And if the user's do need fix it, they can add try-catch block to avoid throw JsonParseException. 

This information is quite useful for problem analysis and have modified many times.